docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,29 @@
+**********************************
+Creating color RGB images
+**********************************
+
+`Lupton et al. (2004)`_ describe an "optimal" algorithm for producing red-green-
+blue composite images from three separate high-dynamic range arrays. This method
+is implemented in `~astropy.visualization.lupton_rgb` as a set of classes
+providing different scalings and a convenience wraper function. To generate a
+color PNG file with the default (arcsinh) scaling:
+
+.. doctest-skip::
+
+>>> import numpy as np
+>>> from astropy.visualization import lupton_rgb
```

Comment:
The `astropy.visualization.lupton_rgb` module isn't part of the official API.
Only the `astropy.visualization.make_lupton_rgb` function is, so please change to:
```python
from astropy.visualization import make_lupton_rgb
```